core/src/services/transitions/cssTransitions.ts

```diff
@@ -26,6 +26,12 @@ export function getTransitionDurationMs(element: HTMLElement): number {
 
 export type CSSTransitionFn = (element: HTMLElement, direction: 'show' | 'hide', animation: boolean, context: object) => void | (() => void);
 
+/**
+ * Create a simple css transition.
+ *
+ * @param start - a function that creates the css animation and returns a clean-up function
```

Comment:
`@param start - {@link CSSTransitionFn} function that creates the css animation and returns a clean-up function`